### PR TITLE
Batch update

### PR DESCRIPTION
**Pull request by thomasopsomer**
As the title says, just a simple method to update records in batch :)

And line ending should be ok this time :)

### CODE REVIEW

File: airtable/airtable.py [hunk 1]
```diff
@@ -435,6 +435,26 @@ def update(self, record_id, fields, typecast=False):
             record_url, json_data={"fields": fields, "typecast": typecast}
         )
 
+    def batch_update(self, records, typecast=False):
+        """
+        Updates a records by their record id's in batch.
+
+        Args:
+            records(``dict``): List of dict: [{"id": record_id, "field": fields_to_update_dict}]
```
Comment (gtalarico) [inline airtable/airtable.py:443]: Could you add missing `Arg` documentation?
https://github.com/gtalarico/airtable-python-wrapper/blob/d829743342c34e11e1fb38fc5579a0533af4900a/airtable/airtable.py#L524

**Review by gtalarico — CHANGES_REQUESTED** [review]
Looks great, thank you for the PR!
Just need `Arg` change on docstring

**Comment by thomasopsomer** [comment]
Sorry I forgot about this... just fixed the docstring

**Review by gtalarico — APPROVED** [review]
LG, thanks!
We should probably include "Default is `False" in doc string but we can fix this in sep. PR

**Comment by DylanScotney** [comment]
Hey @gtalarico do you plan on doing a new release to include this? Would be great to have it as part of the pypi library!

**Comment by gtalarico** [comment]
Thanks for the bump @DylanScotney 
released as .15.12
https://pypi.org/project/airtable-python-wrapper/#history

**Pull request merged**